packages/ui/src/styles/layout.css

```diff
@@ -67,5 +67,5 @@
 }
 
 body {
-  --wcm-z-index: 1010; /* Wallet Connect has an option to override its theme, but this object is not exposed in Onboard, so the override is handled in the UI library. See https://docs.reown.com/advanced/walletconnectmodal/theming for more overrides*/
+  --wcm-z-index: 1301; /* Wallet Connect has an option to override its theme, but this object is not exposed in Onboard, so the override is handled in the UI library. See https://docs.reown.com/advanced/walletconnectmodal/theming for more overrides*/
```

Comment:
WalletConnect was showing behind the modal right?

Reply:
This is an existing issue in prod already with onboard yeah.